app/controllers/admin/users_controller.rb

```diff
@@ -47,11 +47,16 @@ def user_params
   end
 
   def get_users
-    @users = User.by_user_name(params[:name])
-                 .by_status(params[:status])
-                 .by_role(params[:role])
-                 .order_by_post_count(params[:order_by])
-                 .order_created_at
-                 .page(params[:page]).per Settings.user.page
+    @q = User.ransack params[:q]
+    @users = @q.result.page(params[:page]).per Settings.user.page
+  end
+
+  def check_params
+    unless params[:q].present? && (params[:q][:higher].eql? Settings.not_higher)
+      return
+    end
```

Comment:
```suggestion
    return unless params[:q].present? &&
                  (params[:q][:higher].eql? Settings.not_higher)
```
e thử coi xem được không ? =))

Reply:
de em thu